goent.mod

```diff
@@ -37,23 +40,29 @@ require (
 	github.com/prometheus/tsdb v0.7.1
 	github.com/rogpeppe/go-internal v1.5.2 // indirect
 	github.com/rs/cors v1.7.0
-	github.com/sirupsen/logrus v1.6.0
-	github.com/spf13/cast v1.3.0
-	github.com/spf13/pflag v1.0.5 // indirect
-	github.com/spf13/viper v1.6.1
-	github.com/stretchr/objx v0.2.0 // indirect
+	github.com/sirupsen/logrus v1.7.0
+	github.com/spf13/cast v1.3.1
+	github.com/spf13/viper v1.7.1
 	github.com/stretchr/testify v1.6.0
+	github.com/sykesm/zap-logfmt v0.0.4 // indirect
 	github.com/syndtr/goleveldb v1.0.1-0.20190923125748-758128399b1d
 	github.com/tidwall/gjson v1.3.5
 	github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5
 	github.com/urfave/cli v1.22.1
 	github.com/wasmerio/go-ext-wasm v0.3.1
 	github.com/willf/bitset v1.1.11 // indirect
 	github.com/willf/bloom v2.0.3+incompatible
-	go.uber.org/atomic v1.6.0
-	google.golang.org/grpc v1.27.1
+	go.uber.org/atomic v1.7.0
+	go.uber.org/multierr v1.6.0 // indirect
+	go.uber.org/zap v1.16.0 // indirect
+	golang.org/x/crypto v0.0.0-20201124201722-c8d3bf9c5392 // indirect
+	golang.org/x/sys v0.0.0-20201119102817-f84b799fce68 // indirect
+	google.golang.org/genproto v0.0.0-20201119123407-9b1e624d6bc4 // indirect
+	google.golang.org/grpc v1.33.2
 )
 
 replace github.com/golang/protobuf v1.4.2 => github.com/golang/protobuf v1.3.2
 
+replace github.com/hyperledger/fabric => github.com/hyperledger/fabric v2.0.1+incompatible
```

Comment:
why replace fabric

Reply:
some package use different version of this one, will report error if not repace.